src/spotify_dl.py

```diff
@@ -289,7 +296,9 @@ def process_input_url(url: str, interactive: bool) -> list:
             print(f"\t[!] Song not found{f' at {url}' if not interactive else ''}.")
             return []
 
-        track_title = _get_track_local_title(track_resp_json['metadata']['title'], track_resp_json['metadata']['artists'])
+        track_title = assemble_track_custom_title(track_resp_json['metadata']['title'], track_resp_json['metadata']['artists'], 0, filename_template)
```

Comment:
you mind making these kwargs?

Reply:
You mean making it 'assemble_track_custom_title(track_resp_json['metadata']['title'], track_resp_json['metadata']['artists'], **kwargs)' or something else?

Reply:
sorry I mean instead of 
```python
assemble_track_custom_title(track_resp_json['metadata']['title'], track_resp_json['metadata']['artists'], 0, filename_template)
```
use
```python
assemble_track_custom_title(
    title=track_resp_json['metadata']['title'],
    artist=track_resp_json['metadata']['artists'],
    track_num=0,
    template=filename_template
)
```
so it's a little clearer